src/platform/monitoring/frontend-metrics/whitelisted-paths.js

```diff
@@ -7,6 +7,6 @@
 export const whitelistedPaths = [
   '/',
   '/disability/',
-  '/facilities/',
+  '/find-locations/',
```

Comment:
Make sure that the comment above here is addressed.

Reply:
Addressed in this PR, https://github.com/department-of-veterans-affairs/vets-api/pull/2651.